### PR TITLE
move helm version to a group var.

### PR DESCRIPTION
**Pull request by julialongtin**

### CODE REVIEW

File: vpc/ansible/group_vars/helm_version.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+helm3_version: 'v3.1.1'
```
Comment (lucendio) [inline vpc/ansible/group_vars/helm_version.yml:1]: Of course, we wont go backwards, but there will most likely be another major version release for Helm some day, so it probably makes more sense to call the var `helm_version` ;)

Comment (jschaul) [reply vpc/ansible/group_vars/helm_version.yml:1]: There is also a `helm2_version` which specifies the exact version of helm 2 in other parts of the code. This particular change in this PR sounds fine to me, especially if in a migration phase you need both helm2 and helm3.

**Comment by lucendio** [comment]
> There is also a `helm2_version`

Okay, didn't notice that. Then, don't mind me. or my *change of request*

**Comment by jschaul** [comment]
You can dismiss the "request for changes" like this:

dismiss

Or, alternatively, you can click "review" and then "approve", which also cancels your previous request for changes.

**Pull request merged**